File: carGame.js
// Simple Car Game in JavaScript - Pedro Mantovani

class Car {
    constructor(model) {
        this.model = model;
        this.speed = 0;
    }

    accelerate() {
        this.speed += 10;
        console.log(`${this.model} acelerou para ${this.speed} km/h!`);
    }

    brake() {
        this.speed -= 10;
        if (this.speed < 0) this.speed = 0;
        console.log(`${this.model} freou para ${this.speed} km/h!`);
    }
}

// Simulação de uso do carro
const myCar = new Car("McLaren JS");
console.log("Bem-vindo ao Simple Car Game em JS!");

myCar.accelerate();
myCar.accelerate();
myCar.brake();

console.log("Fim do jogo em JS.");

// Para rodar no navegador, precisaria de um arquivo HTML:
/*
<!DOCTYPE html>
<html>
<head>
    <title>Car Game JS</title>
</head>
<body>
    <script src="carGame.js"></script>
</body>
</html>
*/

File: CarGame.cs
using System;

namespace SimpleCarGame
{
    public class Car
    {
        public string Model { get; set; }
        public int Speed { get; set; }

        public Car(string model)
        {
            Model = model;
            Speed = 0;
        }

        public void Accelerate()
        {
            Speed += 10;
            Console.WriteLine($"{Model} acelerou para {Speed} km/h!");
        }

        public void Brake()
        {
            Speed -= 10;
            if (Speed < 0) Speed = 0;
            Console.WriteLine($"{Model} freou para {Speed} km/h!");
        }
    }

    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Bem-vindo ao Simple Car Game!");
            Car myCar = new Car("Fusca Tunado");

            myCar.Accelerate();
            myCar.Accelerate();
            myCar.Brake();

            Console.WriteLine("Fim do jogo.");
        }
    }
}
